### DISCUSSION TITLE
Unable to connect via SSH to github.com

### DISCUSSION THREAD

**Discussion by massix** [community/community#75696] [Other Feature Feedback, Questions, & Ideas]
Hello everyone,
I don't know if this is the right place to ask for help, but since this morning I'm having a weird issue and I think it might be related with GitHub, although the [status page](https://www.githubstatus.com/) does not detect any outage.

If I try to pull my repository using the original `origin` remote, it fails with the following error:
```sh
$ git pull origin main
ssh: connect to host github.com.eastus.cloudapp.azure.com port 22: Connection refused
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
``` 

So of course I tried using `ssh` with all the verbosity on and I could not detect any issue, just a very early connection refused.

I tried connecting via `ssh` directly to the IP address and...

```sh
$ dig github.com -t A
A github.com. 1m00s   140.82.121.4
$ ssh -i ~/.ssh/id_rsa git@140.82.121.4
PTY allocation request failed on channel 0
Hi massix! You've successfully authenticated, but GitHub does not provide shell access.
Connection to 140.82.121.4 closed.
```

It works! I can even `git pull` and `fetch` and everything if I configure a different origin using the plain IP address.

I'm pretty sure it's not my SSH key, and everything was working until 8 to 9 hours ago.. What could be the issue?

**Comment by Nikola-1085** [comment]
****

**Reply by Nikola-1085** [reply]
[]()

**Comment by Sofitt** [comment]
Same. Did you fix this?
